packages/x-components/src/x-modules/identifier-results/store/__tests__/actions.spec.ts

```diff
@@ -159,9 +159,6 @@ describe('testing identifier results module actions', () => {
       await store.dispatch('saveOrigin', { location: 'predictive_layer' });
       expect(store.state.origin).toBeNull();
 
-      await store.dispatch('saveOrigin', { feature: 'search_box' });
```

Comment:
Why did you remove it? Please check the result of this case.

Reply:
I removed it because right now It makes no sense because you can create an origin with only the feature and this test is to check when it's null.

Reply:
But maybe you can create another test but not remove this validation right?

Now you have three different scenarios when you dispatch the action:

- Location
- Feature
- Location + Feature

But you're only testing two of them right?